src/neuroconv/utils/json_schema.py

```diff
@@ -70,104 +69,93 @@ def get_base_schema(
     return base_schema
 
 
-def get_schema_from_method_signature(method: Callable, exclude: list = None) -> dict:
+def get_schema_from_method_signature(method: Callable, exclude: Optional[List[str]] = None) -> dict:
+    """Deprecated version of `get_json_schema_from_method_signature`."""
+    message = (
+        "The method `get_schema_from_method_signature` is now named `get_json_schema_from_method_signature`."
+        "This method is deprecated and will be removed after January 2025."
+    )
+    warnings.warn(message=message, category=DeprecationWarning, stacklevel=2)
+
+    return get_json_schema_from_method_signature(method=method, exclude=exclude)
+
+
+def get_json_schema_from_method_signature(method: Callable, exclude: Optional[List[str]] = None) -> dict:
     """
-    Take a class method and return a json-schema of the input args.
+    Get the equivalent JSON schema for a signature of a method.
+
+    Also uses `docstring_parser` (NumPy style) to attempt to find descriptions for the arguments.
 
     Parameters
     ----------
-    method: function
-    exclude: list, optional
+    method : callable
+        The method to generate the JSON schema from.
+    exclude : list of str, optional
+        List of arguments to exclude from the schema generation.
+        Always includes 'self' and 'kwargs'.
```

Comment:
```suggestion
        Always includes 'self' and 'cls'.
```

Reply:
Good catch - this was copied from the old docstring